prusti-launch/src/bin/prusti-rustc.rs

```diff
@@ -36,8 +36,11 @@ fn process(mut args: Vec<String>) -> Result<(), i32> {
     let libjvm_path = prusti_launch::find_libjvm(&java_home)
         .expect("Failed to find JVM library. Check JAVA_HOME");
 
-    let prusti_sysroot = prusti_launch::prusti_sysroot()
-        .expect(&format!("Failed to find Rust's sysroot for Prusti"));
+    let prusti_sysroot = match env::var("PRUSTI_SYSROOT") {
+        Ok(prusti_sysroot) => PathBuf::from(prusti_sysroot),
+        Err(_) => prusti_launch::prusti_sysroot()
+            .expect(&format!("Failed to find Rust's sysroot for Prusti"))
+    };
```

Comment:
Could you move this match into `prusti_sysroot`?

Reply:
Done in https://github.com/viperproject/prusti-dev/pull/464/commits/856f94c9ecbc019d493007a04f108fe6413044d1, please check if it is ok.